server/handlers/project_command_output_handler.go

```diff
@@ -142,6 +155,32 @@ func (p *AsyncProjectCommandOutputHandler) Handle() {
 	}
 }
 
+// Sets the OperationComplete to true for the job
+func (p *AsyncProjectCommandOutputHandler) updateOutputBufferOperationStatus(jobID string) {
```

Comment:
nit: Looks like you are just setting the value to `true`. Maybe instead of `updateOutputBufferOperationStatus` call it `completeJob`? I don't think you need to be verbose with buffers and statuses.

Reply:
Done!